Code/app_vars.py

```diff
@@ -87,3 +88,48 @@ def get_game_path(cls) -> Optional[Path]:
             return
 
         return game_path
+
+    @classmethod
+    def get_mods_path(cls) -> None:
+        path = Path.home()
+
+        if not path.exists():
```

Comment:
For what?

Reply:
Fair enough, I forgot to remove this when was trying to make "anti-pirate" validation.

Reply:
Actually, the code itself already prevents pirates from using the program.
I doubt they have an AppData directory with mods installed from the workshop. Do they?

Reply:
Yes. In fact, they don't have and load everything via localmods, but localmods can work very crookedly now. (They most likely don't even load into inactive)